pep-0570.rst

```diff
@@ -541,15 +632,33 @@ definition::
    Traceback (most recent call last):
      File "<stdin>", line 1, in <module>
    TypeError: combined_example() takes 2 positional arguments but 3 were given
+
    >>> combined_example(1, 2, kwd_only=3)
    1 2 3
+
    >>> combined_example(1, standard=2, kwd_only=3)
    1 2 3
+
    >>> combined_example(pos_only=1, standard=2, kwd_only=3)
    Traceback (most recent call last):
      File "<stdin>", line 1, in <module>
    TypeError: combined_example() got an unexpected keyword argument 'pos_only'
 
+-----
+Recap
+-----
+
+The use case will determine which parameters to use in the function definition::
+
+   def f(pos1, pos2, /, pos_or_kwd, *, kwd1, kwd2):
+
+As guidance:
+
+* Use positional-only if names do not matter or have no meaning, and there are
+  only a few arguments which will always be passed in the same order.
+* Use keyword-only when names have meaning and the function definition is
+  more understandable by being explicit with names.
+* Consider positional-only if performance is critical and would be optimized.
```

Comment:
I'm not sure I would even mention perf here. Surely this only matters for C functions and even so only in rare cases.